examples/react-router/client/main.jsx

```diff
@@ -5,18 +5,28 @@ import { Meteor } from 'meteor/meteor';
 import { Random } from 'meteor/random';
 import { render } from 'react-dom';
 import { Router, Route, browserHistory, Link } from 'react-router';
+import { Helmet } from "react-helmet";
 
-import OKGAnalytics, { analytics } from '@okgrow/auto-analytics';
+import { initAnalytics } from '@okgrow/auto-analytics';
 
 import './main.html';
+// NOTE: import your own custom built segment analytics.js like below.
+// You can find out how to build your analytics.js here https://github.com/okgrow/analytics.js
+// In this example our analytics.js is bundled with only Google Analytics & Mixpanel
+import analytics from '../imports/analytics.min';
 
-const SETTINGS = (Meteor.settings
+const integrations = (Meteor.settings
                   && Meteor.settings.public
                   && Meteor.settings.public.analyticsSettings)
                   || false;
 
-// Initialize the @okgrow/auto-analytics package
-OKGAnalytics(SETTINGS);
+// NOTE: Initialize the @okgrow/auto-analytics package.
+initAnalytics({ analytics, integrations, options: {}, autorun: true });
+
+const exampleTrackEvent = () => analytics.track('Bought a Ticket', {
```

Comment:
So this is going directly to the analytics package in this case and not ours. Which is cool. Just making sure I understand.

Reply:
I guess maybe it should be or could be `window.analytics` as we assign it inside `intitAnalytics` to the window. I don't think the behaviour has changed in this PR. We always called the `analytics` object, its just we never had an example of `.track()` here. 

The main change this PR has is that we don't provide a default `analytics` bundle. The user has to go build it themselves. Created a bundle for the example to demonstrate the app :)